### DISCUSSION TITLE
Atmosphere libs dependency

### DISCUSSION THREAD

**Discussion by monstrosities** [o0Zz/sys-con#51] [General]
Is it possible to remove Atmosphere libs dependency in this version like was done here:
https://github.com/cathery/sys-con/commit/e79be831614952aa84f7feb48cea5b6329ff07dd

This would mean only one version is needed and the sysmodule won't break as frequently as it won't always need an update on new Atmosphere versions.

**Comment by o0Zz** [comment]
Hi thank you for this share. 
Honestly, I haven’t yet had a chance to dive deeply into Atmosphere and fully understand everything it involves. I agree, though—if we can remove the dependency on Atmosphere, that would be ideal. I’ll take a closer look, but I’m not entirely sure what’s involved, especially since this fork introduces more changes and likely additional dependencies on Atmosphere. I’ll definitely keep this in mind for future releases. Thanks again!
